xc/device.ml

```diff
@@ -61,6 +61,14 @@ module Profile = struct
   let of_domid x = if is_upstream_qemu x then Qemu_upstream else Qemu_trad
 end
 
+(** Represent an IPC endpoint *)
+module Socket = struct
+  type port = Unix of string | Network of int
```

Comment:
Maybe `Socket.t` would be a better type because `port` implies a number (at least for me).

Reply:
done on 4457f655e50b663954f616687886e9f0d20b5292